### PR TITLE
Quickstart update

### PR DESCRIPTION
**Pull request by dnola**
FP16 was only set inside network, not on the input so was suboptimal. Added io format flags to fix this. Additionally, added image examples to make verification of results easier.

### CODE REVIEW

File: quickstart/IntroNotebooks/0. Running This Guide.ipynb [hunk 1]
```diff
@@ -69,7 +69,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pip install pycuda onnx "
+    "!pip install pycuda onnx scikit-image"
```
Comment (rajeevsrao) [inline quickstart/IntroNotebooks/0. Running This Guide.ipynb:72]: Lets use pip3 to be explicit.

File: quickstart/IntroNotebooks/0. Running This Guide.ipynb [hunk 2]
```diff
@@ -195,7 +195,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "version": "3.6.9"
```
Comment (rajeevsrao) [inline quickstart/IntroNotebooks/0. Running This Guide.ipynb:198]: Why did python version change if using the same container?

File: quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb [hunk 3]
```diff
@@ -218,174 +575,159 @@
...
+      "[04/06/2021-04:19:19] [I] Export output to JSON file: \n",
+      "[04/06/2021-04:19:19] [I] Export profile to JSON file: \n",
+      "[04/06/2021-04:19:19] [I] \n",
       "----------------------------------------------------------------\n",
       "Input filename:   resnet50_pytorch.onnx\n",
       "ONNX IR version:  0.0.6\n",
       "Opset version:    9\n",
       "Producer name:    pytorch\n",
-      "Producer version: 1.8\n",
+      "Producer version: 1.6\n",
```
Comment (rajeevsrao) [inline quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb:638]: Torch version also changed?

**Comment by dnola** [comment]
Hello, I also had to fix the Tensorflow conversion path (keras2onnx has been broken: https://github.com/onnx/keras-onnx/issues/651) - and timing of fix is unclear, so moved to tf2onnx. I also updated the versions. Can you take a look?

Thank you!

File: quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb (hunk 3 above)
Comment (dnola) [reply quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb]: I updated to 1.9 to verify this works with latest 21.05 NGC containers. Is that okay?

Comment (rajeevsrao) [reply quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb]: Yes, thanks.

**Review by rajeevsrao — APPROVED** [review]

**Pull request merged**